client/src/pages/Config.tsx

```diff
@@ -0,0 +1,136 @@
+import React, {useEffect, useState} from 'react';
+import { ConfigTypesI, FieldI} from "./ConfigTypes";
+import {
+    Button,
+    Container,
+    Grid, Icon, Paper,
+    Table, TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow, TextField,
+    Typography
+} from "@mui/material";
+import DeleteIcon from '@mui/icons-material/Delete';
+import AddCircleOutlineIcon from '@mui/icons-material/AddCircleOutline';
+import SaveIcon from '@mui/icons-material/Save';
+import EditIcon from '@mui/icons-material/Edit';
+import {APIConfigPath, APIPath} from "../const";
+
+const EmptyField = {
+    name: '',
+    placeHolder: '',
+    disabled: "false",
```

Comment:
Could we use a boolean instead of a string?

Reply:
We could parse it once we get the data from db. And in conditions use it as field.disabled